index.ts

```diff
@@ -0,0 +1,3 @@
+export { createButton as Button } from "./stories/Button"
```

Comment:
nit: we can merge those two export lines right?

Reply:
+1

Reply:
oof, this one. i worked for a solid day on why these exports only worked when separated. let me try to find the article to support—-this is the only solution that i could get to work, which i know is really not ideal. 

are we OK on punting this one for now? i can try to see if there's alternatives, but combining them makes both inacessible on the frontend